### PR TITLE
Norton added. Fixed Mxtoolbox API error

### LINKED ISSUE #154: [New Feature] Norton Check
Can we have this? Still unsure.
image

https://safeweb.norton.com

### PR DESCRIPTION
**Pull request by rachejazz**
Fixes #154 

#### Resolves
Mxtoolbox API error to verbose

#### Changes

-
-
-

#### Checklist

- [x] I have read the Contribution & Best practices Guideline.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The acceptance, integration, unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by rachejazz** [comment]
```
┬─[divya at racharch in ~/a/g/OWASP-Maryam]─[G:norton]─[B:90%, 01:15:39 remaining]
╰──> λ ./maryam -e domain_reputation -d zenithsecurity.org
[*] [Barracuda] Searching in barracuda...
[*] [Multirbl] Searching in multirbl.valli.org...
[*] [MXTOOLBOX] Searching in mxtoolbox...
[!] [domain_reputation:mxtoolbox] Mxtoolbox needs a valid API key.
[*] [NORTON] Scanning domain ...
[*] CATEGORY
[*] 	business

[*] NUMBER OF PRESENCE ON BLACKLIST
[*] 	0

[*] BLACKLISTS
[*] Without result
[*] ABSENCE % ON BLACKLIST
[*] 	100.0

[*] NORTON SAFE WEB REPORT:
[*] 	Threat Report: zenithsecurity.org is a known dangerous web page

┬─[divya at racharch in ~/a/g/OWASP-Maryam]─[G:norton]─[B:89%, 01:23:59 remaining]
╰──> λ ./maryam -e domain_reputation -d owasp.org
[*] [Barracuda] Searching in barracuda...
[*] [Multirbl] Searching in multirbl.valli.org...
[*] [MXTOOLBOX] Searching in mxtoolbox...
[!] [domain_reputation:mxtoolbox] Mxtoolbox needs a valid API key.
[*] [NORTON] Scanning domain ...
[*] CATEGORY
[*] 	computing-technology

[*] NUMBER OF PRESENCE ON BLACKLIST
[*] 	5

[*] BLACKLISTS
[*] 	RFC-Clueless (RFC²) Metalist RBL
[*] 	Woody's SMTP Blacklist URIBL
[*] 	Swinog URIBL
[*] 	RFC-Clueless (RFC²) postmaster RBL
[*] 	RFC-Clueless (RFC²) DSN RBL
[*] ABSENCE % ON BLACKLIST
[*] 	90.9090909090909

[*] NORTON SAFE WEB REPORT:
[*] 	Threat Report: found no issues with owasp.org
```

File: modules/osint/domain_reputation.py [hunk 1]
```diff
@@ -149,6 +170,7 @@ def module_api(self):
 	if LISTS!=0:
 		presence = len(list(set(BLACKLIST))) / LISTS
 		OUTPUT['absence % on blacklist'] = 100 - (presence*100)
+	OUTPUT['Norton Safe Web Report:'] = RESULT
```
Comment (saeeddhqan) [inline modules/osint/domain_reputation.py]: wow. "Safe Web Report:" as a key? change it to safe_web_report.

File: modules/osint/domain_reputation.py [hunk 2]
```diff
@@ -149,6 +170,7 @@ def module_api(self):
 	if LISTS!=0:
 		presence = len(list(set(BLACKLIST))) / LISTS
 		OUTPUT['absence % on blacklist'] = 100 - (presence*100)
```
Comment (saeeddhqan) [inline modules/osint/domain_reputation.py]: absenc_on_blacklists

File: modules/osint/domain_reputation.py [hunk 3]
```diff
@@ -24,10 +24,10 @@
 	'author': 'Kunal Khandelwal',
 	'version': '0.1',
```
Comment (saeeddhqan) [inline modules/osint/domain_reputation.py]: change the version

File: modules/osint/domain_reputation.py (hunk 2 above)
Comment (rachejazz) [reply modules/osint/domain_reputation.py]: Should I change that? That was done by kunal. Sending a commit.

File: modules/osint/domain_reputation.py (hunk 1 above)
Comment (rachejazz) [reply modules/osint/domain_reputation.py]: Eh...sorry, see how the output is printed? Wouldn't it look bad with the __ ?

File: modules/osint/domain_reputation.py (hunk 2 above)
Comment (saeeddhqan) [reply modules/osint/domain_reputation.py]: yes that's missed

File: modules/osint/domain_reputation.py (hunk 3 above)
Comment (rachejazz) [reply modules/osint/domain_reputation.py]: @saeeddhqan Will the underscores be ok then?
```
[*] ABSENCE_%_ON_BLACKLIST
[*] 	100.0

[*] NORTON_SAFE_WEB_REPORT:
[*] 	Threat Report: zenithsecurity.org is a known dangerous web page
```

File: modules/osint/domain_reputation.py (hunk 1 above)
Comment (saeeddhqan) [reply modules/osint/domain_reputation.py]: You can change the view in module_run but not in the json output

File: modules/osint/domain_reputation.py (hunk 3 above)
Comment (saeeddhqan) [reply modules/osint/domain_reputation.py]: put the % in value instead of key. the key should be exactly like a username

**Comment by rachejazz** [comment]
@saeeddhqan I think working on a different branch for #187 will be better :)
You can consider this for merging

**Pull request merged**